projects/program-website/src/app/core/services/firebase.service.ts~HEAD

```diff
@@ -1,4 +1,3 @@
-import { Injectable } from '@angular/core';
```

Comment:
Not just first string. Whole file.
Look at the name:
```
firebase.service.ts~HEAD
```
Also, we already have firebase service.
So, can you delete the file?